src/anomalib/deploy/inferencers/openvino_inferencer.py

```diff
@@ -37,11 +37,12 @@ class OpenVINOInferencer(Inferencer):
         metadata (str | Path | dict, optional): Path to metadata file or a dict object defining the
             metadata.
             Defaults to ``None``.
-        device (str | None, optional): Device to run the inference on.
-            Defaults to ``CPU``.
+        device (str | None, optional): Device to run the inference on (AUTO, CPU, GPU, NPU).
+            Defaults to ``AUTO``.
         task (TaskType | None, optional): Task type.
             Defaults to ``None``.
-
+        config (dict | None, optional): Configuration parameters for the inference
```

Comment:
Would we still need the `config` in the new approach? We ideally would like to get rid of `config` since the dictionary hides the arguments in the class signature.

Reply:
I don't want to break the current behavior in this PR. Reworking inference config is on my list for future work.